dll/win32/shell32/lang/de-DE.rc

```diff
@@ -770,17 +770,17 @@ BEGIN
     IDS_PASTE "Einfügen"
     IDS_EJECT "Auswerfen"
     IDS_DISCONNECT "Trennen"
-    IDS_OPENFILELOCATION "Open f&ile location"
-    IDS_DESKLINK "Desktop (Create shortcut)"
-    IDS_SENDTO_MENU "Se&nd To"
-
-    IDS_MOVEERRORTITLE "Error Moving File or Folder"
-    IDS_COPYERRORTITLE "Error Copying File or Folder"
-    IDS_MOVEERRORSAMEFOLDER "Cannot move '%s': The destination folder is the same as the source folder."
-    IDS_MOVEERRORSAME "Cannot move '%s': The source and destination file names are the same."
-    IDS_COPYERRORSAME "Cannot copy '%s': The source and destination file names are the same."
-    IDS_MOVEERRORSUBFOLDER "Cannot move '%s': The destination folder is a subfolder of the source folder."
-    IDS_COPYERRORSUBFOLDER "Cannot copy '%s': The destination folder is a subfolder of the source folder."
+    IDS_OPENFILELOCATION "Dateipfad öffnen"
+    IDS_DESKLINK "Desktop (Verknüpfung erstelln)"
+    IDS_SENDTO_MENU "Senden an"
+
+    IDS_MOVEERRORTITLE "Fehler beim Verschieben der Datei/des Ordners"
+    IDS_COPYERRORTITLE "Fehler beim Kopieren der Datei/des Ordners"
+    IDS_MOVEERRORSAMEFOLDER "'%s'kann nicht verschoben werden: Der Zielordner ist derselbe Ordner wie der Quellordner."
```

Comment:
Write the sentence like you did at IDS_MOVEERRORSAME.
I would write 'identisch' instead of 'gleich'